ports/espressif/peripherals/i2c.c

```diff
@@ -35,7 +35,7 @@ typedef enum {
 static i2c_status_t i2c_status[I2C_NUM_MAX];
 
 void i2c_reset(void) {
-    for (i2c_port_t num = 0; num < I2C_NUM_MAX; num++) {
+    for (i2c_port_t num = 0; num < (int)I2C_NUM_MAX; num++) {
```

Comment:
`int` not a `u8` or something unsigned at least?

Reply:
Switching to `(i2c_port_t)` so they match.